artemis-server/src/main/resources/schema/artemis-configuration.xsd

```diff
@@ -54,6 +54,15 @@
             </xsd:annotation>
          </xsd:element>
 
+         <xsd:element name="amqp-treat-reject-as-unmodified-delivery-failed" type="xsd:boolean" maxOccurs="1" minOccurs="0" default="false">
```

Comment:
If you use the protocol manager (AMQP Protocol Manager, such property would just be part of BeanUtils, and you wouldn't need to change any schema.

Reply:
Done